### DISCUSSION TITLE
Get peaks name according to the CoveragePlot

### DISCUSSION THREAD

**Discussion by yuhanH** [stuart-lab/signac#419] [Q&A] [answered]
Hi, 
I am wondering how I can get the highlighted peaks name from the CoveragePlot.

![image](https://user-images.githubusercontent.com/29529544/106337351-f4e49100-625e-11eb-8228-784431a76931.png)
Thanks!

**Comment by timoast** [comment, marked as answer]
In many cases you can treat a Seurat object with a ChromatinAssay as the active assay like a `GRanges` object, and use functions in the `GenomicRanges` package for interacting with the ranges (see [here](https://satijalab.org/signac/articles/data_structures.html#other-chromatinassay-methods-1) for more information). In this case you can find overlaps between the region of interest and the peaks to get the index of the overlapping peaks, then extract the actual ranges of those peaks, for example:

```r
library(Signac)
library(GenomicRanges)

# obj is an example Seurat object containing a ChromatinAssay

overlap_peaks <- findOverlaps(obj, StringToGRanges("chr1-10000000-10010000"))
granges(obj)[queryHits(overlap_peaks)]
```

In this particular case, the region you highlighted in the box is part of a peak that seems to contain two distinct regions, so you could try adjusting the peak calling parameters to try to pick these up as two separate regions if you really want to find the coordinates of just that highlighted region.
